src/adapter/threads.ts

```diff
@@ -139,6 +140,9 @@ const sourcesEqual = (a: Dap.Source, b: Dap.Source) =>
   a.sourceReference === b.sourceReference &&
   urlUtils.comparePathsWithoutCasing(a.path || '', b.path || '');
 
+const getReplSourceSuffix = () =>
+  `\n//# sourceURL=eval-${randomBytes(4).toString('hex')}${SourceConstants.ReplExtension}\n`;
```

Comment:
I copied the code from the other suffix version, but this won't work because the golden files contain the string, which is random. Hopefully I can find a way to only have them as a marker and keep the original `VM<xx>` text instead.

Reply:
https://github.com/microsoft/vscode-js-debug/blob/42cfe59d08076168ae0099b78891aacdf792f770/src/test/goldenText.ts#L185-L189

Reply:
So, you'd prefer if I were to just stirp the number like `VM<xx>`?